sdk/burdock/sql/private_reader.py

```diff
@@ -224,3 +224,17 @@ def execute_ast_typed(self, query):
             raise ValueError("Please pass ASTs to execute_typed.  To execute strings, use execute.")
         subquery_results = self._preprocess(query)
         return self._postprocess(*subquery_results)
+
+class PrivateReaderOptions:
+    def __init__(self, 
+        censor_dims=True, 
+        clamp_counts=True, 
+        reservoir_sample=True,
+        clamp_columns=True,
+        row_privacy=False):
+
+        self.censor_dims = censor_dims
```

Comment:
param description?

I noticed some params shared the same in RewriterOptions, is it possible to combine them together?

Reply:
I updated the docstrings.